src/private/appEntity.ts

```diff
@@ -48,14 +48,17 @@ export namespace appEntity {
    *
    * Open the Tab Gallery and retrieve the app entity
    * @param threadId ID of the thread where the app entity will be created
-   * @param categories A list of app categories that will be displayed in the open tab gallery
+   * @param categories A list of app categories that will be displayed in the opened tab gallery
+   * @param launchContext An object that will be made available to the application being configured
+   *                      through the Teams Context's subEntityId field.
    * @param callback Callback that will be triggered once the app entity information is available.
    *                 The callback takes two arguments: the app entity configuration, if available and
    *                 an optional SdkError in case something happened (i.e. the window was closed)
    */
   export function selectAppEntity(
     threadId: string,
     categories: string[],
+    launchContext: string,
```

Comment:
Shouldn't we also update the Context interface that apps will receive this launchContext through?

Reply:
There is no new field added to the Context after the discussions with the MetaOS team. This launchContext param will show up in the subEntityId field.